chapters/VK_EXT_debug_utils.adoc

```diff
@@ -28,9 +28,19 @@ callback to capture debug messages from a variety of Vulkan components.
 
 It can be useful for an application to provide its own content relative to a
 specific Vulkan object.
+
 The following commands allow application developers to associate
 user-defined information with Vulkan objects.
-
+These commands are device-level commands but they may: reference instance-level
+objects (such as slink:VkInstance) and physical device-level objects (such as
+slink:VkPhysicalDevice) with a few restrictions:
+  * The data for the corresponding object may: still be available after the
+    slink:VkDevice used in the corresponding API call to set it is destroyed,
+    but access to this data is not guaranteed and should be avoided.
```

Comment:
You can probably omit `The data for the corresponding object may: still be available after the slink:VkDevice used in the corresponding API call to set it is destroyed, but access to this data is not guaranteed and should be avoided.` - there's no way for an application or layer to query the data anyway, as far as I can tell, and whether a layer or an implementation can still access it in their own data structires is up to how they hang the name off objects.